azext_iot/operations/hub.py

```diff
@@ -181,6 +181,112 @@ def iot_device_delete(cmd, device_id, hub_name=None, resource_group_name=None, l
         raise CLIError(unpack_msrest_error(e))
 
 
+def iot_device_get_parent(cmd, device_id, hub_name=None, resource_group_name=None, login=None):
+    target = get_iot_hub_connection_string(cmd, hub_name, resource_group_name, login=login)
+    child_device = _iot_device_show(target, device_id)
+    if child_device['capabilities']['iotEdge']:
```

Comment:
Why not use one of the new validate helpers below?

Reply:
Fixed